diff --git a/node_modules/conventional-changelog-core/index.js b/node_modules/conventional-changelog-core/index.js
index 435a9d6..43dde01 100644
--- a/node_modules/conventional-changelog-core/index.js
+++ b/node_modules/conventional-changelog-core/index.js
@@ -28,7 +28,7 @@ function conventionalChangelog (options, context, gitRawCommitsOpts, parserOpts,
       ...gitRawCommitsOpts,
       from: from,
       to: to
-    })
+    }, gitRawExecOpts)
       .on('error', function (err) {
         if (!commitsErrorThrown) {
           setImmediate(commitsStream.emit.bind(commitsStream), 'error', err)
diff --git a/node_modules/conventional-changelog-core/lib/merge-config.js b/node_modules/conventional-changelog-core/lib/merge-config.js
index 4c5c26a..7f8c19f 100644
--- a/node_modules/conventional-changelog-core/lib/merge-config.js
+++ b/node_modules/conventional-changelog-core/lib/merge-config.js
@@ -22,7 +22,8 @@ function semverTagsPromise (options) {
       lernaTags: !!options.lernaPackage,
       package: options.lernaPackage,
       tagPrefix: options.tagPrefix,
-      skipUnstable: options.skipUnstable
+      skipUnstable: options.skipUnstable,
+      cwd: options.pkg?.path
     }, (err, result) => {
       if (err) {
         reject(err)
